tests/local/network.go

```diff
@@ -351,7 +350,8 @@ func (n *localNetwork) RelayMessage(ctx context.Context,
 	sourceReceipt *types.Receipt,
 	source interfaces.SubnetTestInfo,
 	destination interfaces.SubnetTestInfo,
-	expectSuccess bool) *types.Receipt {
+	expectSuccess bool,
+) *types.Receipt {
```

Comment:
Might just be the way Github is displaying this, but can you gofmt this file?

Reply:
It's not github, I moved the return type to its own line. All of our other methods do this, but I don't think gofmt/goimports has an opinion on this for whatever reason.